app/views/devise/registrations/new.html.haml

```diff
@@ -1,10 +1,10 @@
 %h2 Sign up
-= simple_form_for(resource, :as => resource_name, :url => registration_path(resource_name)) do |f|
+= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f|
```

Comment:
I think this page is unused as well.

Reply:
Do we use any of the devise views? I might do a separate PR to just take them all out.

Reply:
I think we use the sessions views and possibly one or two of the mailers. Separate PR is fine.